Models/SingletStandardModel_Z2/singletStandardModelZ2.py

```diff
@@ -636,10 +641,10 @@ def configureCollisionIntegration(
 
         inOutCollisionTensor.setIntegrationVerbosity(verbosity)
 
-    def configureManager(self, inOutManager: "WallGo.WallGoManager") -> None:
+    def updateConfig(self, inOutConfig: "WallGo.Config") -> None:
         """Singlet example uses spatial grid size = 20"""
```

Comment:
See comment on wallGoManager.py. I would personally keep this as it was to keep things a bit more direct.